### DISCUSSION TITLE
Php 7.4 minimal symfony version

### DISCUSSION THREAD

**Discussion by yerude** [symfony/symfony#43136] [Q&A] [answered]
Hello,
i would like to upgrade php from 7.1 to 7.4 or even higher and my symfony is only v2.7.

Which minimal symfony version that supports 7.4?
I only find minimal requirements but not when it starts to support specific versions.
On packagist symfony 3.4.49 is listed as php: ^5.5.9|>=7.0.8, but i am a little bit sceptical because of some bad experiences i made in the past.

In this document there is a description what to do for an upgrade:
https://symfony.com/doc/2.3/cookbook/upgrade/major_version.html

Is 3.4.49 the correct version candidate for me?

**Comment by lyrixx** [comment, marked as answer]
https://github.com/symfony/symfony/blob/3.4/.travis.yml

Symfony 3.4 was tested on PHP 7.4 

I checked and the branch 2.* was not. So I guess you got your answer.

**Comment by stof** [comment]
> Is 3.4.49 the correct version candidate for me?

Well, if you are working on upgrading Symfony major versions, I suggest you to not stop at 3.4, which already reached its end of life as well.

Regarding supported PHP versions, the rule is that a Symfony version will support all PHP versions that are released during its lifetime (until the end of bugfix support, not until the end of security support). Symfony 2.7 received bug fixes [until May 2018](https://symfony.com/releases/2.7), meaning that it should have supported PHP 7.2, but not newer versions: https://www.php.net/supported-versions.php
